src/helpers/constraints.js

```diff
@@ -0,0 +1,73 @@
+export var validate = require('validate.js');
```

Comment:
are you using babel or any other compiler? if so try _import validate from 'validate.js';_ instead. Besides the constraints below should be const.